src/services/userServices.js

```diff
@@ -0,0 +1,21 @@
+const angular = require('angular');
+
+angular.module('app-bootstrap').factory('logIn', [
```

Comment:
Since this file is called "userServices" it would be convenient to name the factory "userServices" too, so in the future we can add more requests and methods regarding user functionalities